src/Raven.Client/Json/BlittableOperation.cs

```diff
@@ -223,6 +224,14 @@ private static bool CompareValues(BlittableJsonReaderObject.PropertyDetails oldP
                 return @double % 1 == 0 && @long.Equals((long)@double);
             }
 
+            if (oldProp.Token == BlittableJsonToken.LazyNumber && newProp.Token == BlittableJsonToken.LazyNumber)
+            {
```

Comment:
Can we check the raw string value first, to avoid the allocations?

Reply:
I think it's already done by first condition in 102 line.  It's first comparing length of string and when new and old are equal run Memory.CompareInline method.

Reply:
Correct